packages/datetime/src/datepicker.md

```diff
@@ -6,6 +6,35 @@ A `DatePicker` shows a monthly calendar and allows the user to choose a single d
 
 @reactExample DatePickerExample
 
+@## Shortcuts
+
+The menu on the left of the calendars provides "shortcuts" that allow users to
+quickly select common dates. The items in this menu are controlled through
+the `shortcuts` prop: `true` to show presets, `false` to hide (default), or an
+array of `IDatePickerShortcut` objects to define custom shortcuts.
+
+The **preset shortcuts** can be seen in the example above. They are as follows:
+
+- Today
+- Yesterday
+- 1 week ago
+- 1 month ago
+- 3 months ago
```

Comment:
this is a pretty long list of presets. let's remove this one

Reply:
Hesitant to remove 3 months ago as that is a default that users are already used to in our existing platform where we will be consuming this.

Reply:
I'll remove 6 months ago instead